### ISSUE TITLE
REPL sometimes displays "error", even though the statement is valid

### ISSUE THREAD

**Issue by bmharsha** [JuliaComputing/JuliaDT#11]
This phenomenon only occurs if the previous statement is erroneous.

Example:- 
```
julia>this_("should create an error")
LoadError: UndefVarError: this_ not defined
while loading string, in expression starting on line 1

julia>a = "I am a valid statement"
"error"
I am a valid statement

julia>a = "I am still a valid statement"
I am still a valid statement

julia>
```
In the above code snippet, we  encountered an error in the first statement but the second statement was valid and yet it displayed the string ```error```, this happens to every statement that gets executed after an erroneous statement is encountered, you can observe the third statement went fine.

**Comment by bmharsha** [comment]
I just tested the latest code (Master) on Windows, this issue doesn't occur anymore, one of the latest commit seems to have fixed this issue.

**Comment by snefru** [comment]
Fixed (5accf444247c2d4c37dbb4726c971dd250990b9d)
